modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/security/DoubleSignDetectRunner.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.node.shared.domain.security
+
+trait DoubleSignDetectRunner[F[_]] {
```

Comment:
Haven't you changed the *Runner name?

Reply:
`DoubleSignDetectRunner` has been renamed to `DoubleSignDetect`.